source/serializer/gson.rst

```diff
@@ -11,6 +11,38 @@ An average user of this text serializer will typically want to only deserialize
 component from an external source - serialization is done automatically by the
 :doc:`/platform/index` when the component is sent to the user.
 
+**Importing this serializer into your project**
+
+.. tabs::
+
+   .. group-tab:: Maven
+
+      .. code:: xml
+
+         <dependency>
+            <groupId>net.kyori</groupId>
+            <artifactId>adventure-text-serializer-gson</artifactId>
+            <version>4.7.0</version>
```

Comment:
It might be a good idea to have some sort of checklist somewhere in the repository, of the files that need version numbers updating in.

Reply:
I also thought about this.. maybe there is a feature where you can create some variable that can be updated automatically?

Reply:
On second thought, maybe you can version pages separately? That would solve some problems, because versions on serializers won't always line up with the platforms etc...

Reply:
Finally fixed this